qlty-check/src/parser/radarlint.rs

```diff
@@ -53,11 +62,33 @@ impl Parser for Radarlint {
                             })
                         },
                         |text_range| {
+                            // Convert values to u32, treating -1 as non-existent (use 0 instead)
+                            let start_line = if text_range.start_line < 0 {
+                                0
+                            } else {
+                                text_range.start_line as u32
+                            };
+                            let start_column = if text_range.start_line_offset < 0 {
+                                0
+                            } else {
+                                text_range.start_line_offset as u32
+                            };
+                            let end_line = if text_range.end_line < 0 {
+                                0
+                            } else {
+                                text_range.end_line as u32
+                            };
+                            let end_column = if text_range.end_line_offset < 0 {
+                                0
+                            } else {
+                                text_range.end_line_offset as u32
+                            };
```

Comment:
Can we use `max` to simplify this pattern? If so, let's use that for all four of these assignments. And, let's get rid of the local variables and just put it into the `Range { ... }` constructor